planning/Discovery/work-items.md

```diff
@@ -0,0 +1,51 @@
+     - However, a single list of requirements would allow us to address requirements that span more than one deliverable
+
+### Prioritized List
+1. Direct - [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
+    * Needs to be broken into smaller work-items to better support prioritization.
+2. Direct - [Discovery of Thing Models](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-thing-models-workitem) 
+3. Direct - [Discovery CoAP Directory](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-coap-dir-workitem)
+4. Direct - [Discovery JSON Path Query Language](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-jsonpath-query-language-workitem)
+5. Direct - [Discovery Query Filters](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-query-filters-workitem)
+6. Dependency - [Signing](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#signing-workitem)
+7. Dependency - [Canonicalization](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#canon-workitem)
+8. Dependency - [Normative TD Parsing, Consuming and Validation](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#td-consumption-workitem)
+    * Relevance: In order to support queries, TD content needs to be parsed by directory, and only valid TDs should be stored in TDDs.
+9. Dependency - [Linting mechanism for TDs](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#td-linting-workitem)
+    * As above, but if meant for development may not be relevant to TDDs; to discuss.
+10. Dependency - [Geolocation](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#geolocation-workitem)
+    * Assuming geolocation data added to TDs, this would add geolocation query support to Discovery.
+    * Requirements: https://github.com/w3c/wot-architecture/blob/main/REQUIREMENTS/geolocation.md
+11. Dependency - [Onboarding](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#onboarding-workitem)
+    * Onboarding is actually about establishing trust and identification but may be a preliminary to or related to TDD registration.
+12. Dependency - [TD Versioning](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#td-versioning-workitem)
+    * Impact on Discovery includes how or whether to maintain multiple versions of a single TD, and possibly filtering by version.
+13. Dependency - [Interoperability Profiles](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#profiles-workitem)
+    * Should TDDs also satisfy a Profile?
+14. Process - [WoT CG Coordination](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#wotcg-coordination)
+    * Gathering information on requirements, implementations, feedback on current versions, suggestions for improvement, etc.
+
+## Other/New/Proposed
```

Comment:
One thing I was wondering about is whether there should be some mechanism or framework for Binding Templates to define their own ways of discovering TDs.

E.g., at the moment we have explicit normative statements for HTTP and CoAP in the Discovery specification, but some of them could also be moved to their respective binding documents. Also, if a new binding like OPC-UA emerges, it would not have to make changes to the main discovery document but could define a specific mechanism on its own.

(Sorry if this has already been discussed or is already covered by the existing work items, if so then we can simply mark this as resolved.)